src/Desktop/components/chats/index.jsx

```diff
@@ -97,15 +95,13 @@ function Chats() {
 
                 <ul className="desktop-chats-container">
                     {
-                        chats.map((c, i) => (
-                            <ChatsList
-                                key={v4()}
-                                c={c} />
-                        ))
+                        <ChatsList
+                            key={v4()}
```

Comment:
```suggestion
                            key={key}
```
Usar el valor de key en vez de generar un UUID, para simplificar las cosas